### PR TITLE
Update helm match policy version

### PR DESCRIPTION
**Pull request by wuchang0201**
if kindest/node version < 1.21， the policy Version is different, helm deploy should set policy Version

test report：
image
image

### CODE REVIEW

**Comment by linuzb** [comment]
The elasticsearch-clean-job may have the same problem
https://github.com/alipay/container-observability-service/blob/b7c834f12ac904b8a5778e2a5d19f9c6becfdfb2/deploy/helm/lunettes/templates/elasticsearch/elasticsearch-clean-job.yaml#L1-L2

**Comment by linuzb** [comment]
Could you squash the commit into one?

File: README-zh_CN.md [hunk 1]
```diff
@@ -52,7 +52,11 @@ Lunettes 可以识别容器生命周期每个交付阶段的开始和结尾，
 ```bash
 # install lunettes
 # 该过程开启apiserver的审计功能，apiserver会发生重启
-helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version] 
+helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version]
+
+#如果kindest/node版本 < v1.21，需要设置policyVersion的版本
```
Comment (linuzb) [inline README-zh_CN.md:57]: What should be emphasized here is the kubernetes version rather than the kind version

**Comment by wuchang0201** [comment]
> The elasticsearch-clean-job may have the same problem
> 
> https://github.com/alipay/container-observability-service/blob/b7c834f12ac904b8a5778e2a5d19f9c6becfdfb2/deploy/helm/lunettes/templates/elasticsearch/elasticsearch-clean-job.yaml#L1-L2

File: README-zh_CN.md (hunk 1 above)
Comment (wuchang0201) [reply README-zh_CN.md]: done

**Pull request closed without merging**